File: app/src/main/java/com/paycraft/ticketbooker/models/VerifyOTP.java
package com.paycraft.ticketbooker.models;

import com.google.gson.annotations.SerializedName;

public class VerifyOTP {
    @SerializedName("success")
    private
    boolean isValid;

    public VerifyOTP(boolean isValid) {
        this.isValid = isValid;
    }

    public boolean isValid() {
        return isValid;
    }

    public void setValid(boolean valid) {
        isValid = valid;
    }

    @Override
    public String toString() {
        return "VerifyOTP{" +
                "isValid=" + isValid +
                '}';
    }
}

File: app/src/main/java/com/paycraft/ticketbooker/models/VerifyOTP.kt
package com.paycraft.ticketbooker.models

import com.google.gson.annotations.SerializedName

class VerifyOTP(@field:SerializedName("success")
                var isValid: Boolean) {

    override fun toString(): String {
        return "VerifyOTP{" +
                "isValid=" + isValid +
                '}'.toString()
    }
}
